package.json

```diff
@@ -5,13 +5,17 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "start": "node index.js",
+    "watch": "./node_modules/nodemon/bin/nodemon.js -L index.js"
```

Comment:
You can just use `nodemon -L index.js` node will use the local node_modules first :)